test/test_integration_cluster.rb

```diff
@@ -222,13 +222,19 @@ def test_worker_timeout
   end
 
   def test_idle_timeout
```

Comment:
This updated test fails on master:
<img width="750" alt="Screenshot 2023-11-28 at 7 10 04 pm" src="https://github.com/puma/puma/assets/54629302/cbb40651-69a0-4743-9676-76bee76cec9d">